### PR TITLE
AT_01.01.05 British flag icon is visible

### PR DESCRIPTION
**Pull request by TupakAmaru13**
My first test to verify the British flag icon on the Start page

[TC_01.01.0](https://trello.com/c/CEM5T3no)

### CODE REVIEW

**Review by EkaterinaRomankova — APPROVED** [review]

File: cypress/e2e/testUiAndFunction/startPage/multilanguageSection/US_01.01_Multilanguage_Section_Elements_UA_And_Functionality.cy.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+/// <reference types="cypress"/>
```
Comment (elviraKh1) [inline cypress/e2e/testUiAndFunction/startPage/multilanguageSection/US_01.01_Multilanguage_Section_Elements_UA_And_Functionality.cy.js:1]: US_01.01 already exists

**Review by EkaterinaRomankova — APPROVED** [review]

**Comment by TupakAmaru13** [comment]
Rebased and moved the code around, please review it again @elviraKh1

File: cypress/e2e/testUiAndFunction/startPage/multilanguageSection/US_01.01_MultilanguageSectionElements.cy.js [hunk 2]
```diff
@@ -18,6 +18,10 @@ describe('US_01.01 | Multilanguage section elements UI and functionality', () =>
         .should('be.visible');
     }); 
     
+    it ('TC_01.01.05| Start page >Multilanguage Section > British flag icon is visible', ()=>{
```
Comment (elviraKh1) [inline cypress/e2e/testUiAndFunction/startPage/multilanguageSection/US_01.01_MultilanguageSectionElements.cy.js:21]: in "it" we use  "AT"  and short name

**Review by elviraKh1 — APPROVED** [review]

**Comment by TupakAmaru13** [comment]
Updated test case name @elviraKh1

**Pull request merged**